internal/server/handler.go

```diff
@@ -141,3 +150,31 @@ func (s *Server) writeError(w http.ResponseWriter, sourceErr error) {
 		s.logger.Warn("failed to write error response", mlog.Err(err))
 	}
 }
+
+func (s *Server) validateRequestMatchesInstallationID(r *http.Request, installationID string) error {
+	addr := strings.Split(r.RemoteAddr, ":")[0]
```

Comment:
Do we need to handle IPv6?

Reply:
Good question. At this time, all of our internal k8s networking is IPv4 only. We would have to revisit this if we ever needed IPv6 support.